Makefile.features

```diff
@@ -23,10 +23,11 @@ FEATURES_PROVIDED += cpu_$(CPU)
 
 # Features that are conflicting for all architectures
 FEATURES_CONFLICT += picolibc:newlib
-FEATURES_CONFLICT_MSG += "Only one standard C library can be used"
+FEATURES_CONFLICT_MSG += "Only one standard C library can be used."
 
 FEATURES_CONFLICT += periph_gpio_irq:periph_gpio_ll_irq
-FEATURES_CONFLICT_MSG += "Only one GPIO IRQ implementation can be used"
+FEATURES_CONFLICT_MSG += "Only one GPIO IRQ implementation can be used."
```

Comment:
Shouldn't this belong to another PR ?

Reply:
Hm, it's such a small change that improves readability. Since we are touching the file anyway, I thought we could improve it. Without this small change, the message looks like this:
```
Rationale: Only one standard C library can be used Only one GPIO IRQ implementation can be used Package tinyUSB is not yet compatible with periph_usbdev.
```
That is, the message is completely unstructured.

With this small change, the message looks like this:
```
Rationale: Only one standard C library can be used. Only one GPIO IRQ implementation can be used. Package tinyUSB is not yet compatible with periph_usbdev.
```

Reply:
That is, the problem is that all messages are simply strung together without punctuation.